src/sage/algebras/quatalg/quaternion_algebra.py

```diff
@@ -2464,6 +2631,35 @@ def quadratic_form(self):
         # now get the quadratic form
         return QuadraticForm(gram_matrix)
 
+    def minimal_element(self):
+        r"""
+        Return an element in this quaternion ideal of minimal norm.
+
+        If the ideal is a principal lattice, this method can be used
+        to find a generator.
```

Comment:
It might be helpful to the user to give a reference for this statement, e.g. to Corollary 1.20 in [Piz1980].